test/e2e/case9_templates_test.go

```diff
@@ -35,15 +43,17 @@ var _ = Describe("Test policy templates", func() {
 		It("should resolve templates and propagate to cluster ns managed1", func() {
 			By("Patching test-policy-plr with decision of cluster managed1")
 			plr := utils.GetWithTimeout(
-				clientHubDynamic, gvrPlacementRule, case9PolicyName+"-plr", testNamespace, true, defaultTimeoutSeconds,
+				clientHubDynamic, gvrPlacementRule, case9PolicyName+"-plr", testNamespace,
```

Comment:
What are all these line length changes for? Was the linting failing before?

Reply:
Yes, I had a few lines that were too long and then it bothered me that all the lines didn't look similar. 😆